elements/rh-tabs/rh-tabs.ts

```diff
@@ -192,14 +196,15 @@ export class RhTabs extends LitElement {
 
   override render() {
     const { on = '' } = this;
+    const rtl = this.#dir.dir === 'rtl';
     return html`
-      <div id="rhds-container" class="${classMap({ [on]: !!on })}">
+      <div id="rhds-container" class="${classMap({ [on]: !!on, rtl })}">
         <div part="container" class="${classMap({ overflow: this.#overflow.showScrollButtons })}">
           <div part="tabs-container">${!this.#overflow.showScrollButtons ? '' : html`
             <button id="previousTab" tabindex="-1"
                     aria-label="${this.getAttribute('label-scroll-left') ?? 'Scroll left'}"
                     ?disabled="${!this.#overflow.overflowLeft}"
-                    @click="${() => this.#overflow.scrollLeft()}">
+                    @click="${() => !rtl ? this.#overflow.scrollLeft() : this.#overflow.scrollRight()}">
```

Comment:
test this...

Reply:
@bennypowers PTAL, definitely need a better way to add `dir="rtl"` to the test so im not having to clone and reconnect the component to pick up the ancestor change.

Reply:
i think you can just call connectedCallback again

https://github.com/RedHat-UX/red-hat-design-system/blob/b443fe21700a2a514f9aa24bc291be34e8bd370c/lib/DirController.ts#L16

Reply:
🥇

Reply:
why i just glossed over that such obvious fix /smh 🙃